.gitattributes

```diff
@@ -1,3 +1,7 @@
+# SPDX-License-Identifier: CC0-1.0
+#
+# SPDX-FileCopyrightText: Copyright 2022 Micron Technology, Inc.
```

Comment:
Why not use same license as the python code?  Apache or MIT?

Reply:
I see no reason this shouldn't just be public domain.

Reply:
The reasons are 1) to be consistent, and 2) because legal has recommended Apache 2.0 and MIT.

Reply:
So even readme's should be `Apache-2.0 OR MIT`?

Reply:
The only HSE-related files that must be dual licenced are those which might be included/linked with GPL code.  All of our files to date have been marked Apache 2.0, except for Markdown files since Markdown doesn't support in-file comments -- so those are covered by the top-level LICENSE file (which again to date has been Apache 2.0 only).